.github/workflows/deploy-camunda-definitions.yml

```diff
@@ -34,8 +34,8 @@ jobs:
         with:
```

Comment:
Sonar created lots of code smells. I believe we need to configure to ignore some `.js` files.

Reply:
The code smells(except TODOS) are because of `api-response-parser.js` which was moved form packages to backend.

Reply:
`.js` files must be included. That particular `.js` file has that amount of issues because we were forced to make it ES5 compatible to be executed on the old Camunda 7 javascript engine, which we no longer needed.

Reply:
so, will it be resolved when we do this story https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/1801

Reply:
While the file is in our repo and we move it around Sonarcloud will always review it. When we execute the mentioned ticket, it will be raising again the issues unless, instead of moving it, we delete it.